src/FileSystem/SevenZipArchive.cpp

```diff
@@ -204,7 +204,7 @@ bool CSevenZipArchive::GetFile(unsigned int fid,
 			     &allocImp, &allocTempImp);
 	if (res == SZ_OK) {
 		buffer.resize(outSizeProcessed);
-		memcpy(&buffer[0], (char*)outBuffer + offset, outSizeProcessed);
+		memcpy(buffer.data(), (char*)outBuffer + offset, outSizeProcessed);
```

Comment:
instead checking for emptiness we take invalid pointer and assume memcpy will do nothing with it